From 4114cd3cad83936b8d641d4f04278aea0151e447 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Wed, 19 Oct 2022 15:56:26 +0200
Subject: [PATCH 0310/1695] plat-stm32mp1: conf: introduce CFG_SYSCFG switch

Places stm32mp1_syscfg.c under CFG_SYSCFG switch and default
enable it.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git f6a641ab24bf373357dda5f22f428b8a0eca9834 plat-stm32mp1: conf: introduce CFG_SYSCFG switch]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: If9682647c8bbfb0d9260338c5f48e36eed28bab5
---
 core/arch/arm/plat-stm32mp1/conf.mk        | 1 +
 core/arch/arm/plat-stm32mp1/drivers/sub.mk | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index 71d6435fa..142d0b6a0 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -255,6 +255,7 @@ CFG_STM32_SAES ?= y
 CFG_STM32_TAMP ?= y
 CFG_STM32_UART ?= y
 CFG_STPMIC1 ?= y
+CFG_SYSCFG ?= y
 CFG_TZC400 ?= y
 
 CFG_DRIVERS_I2C ?= $(CFG_STM32_I2C)
diff --git a/core/arch/arm/plat-stm32mp1/drivers/sub.mk b/core/arch/arm/plat-stm32mp1/drivers/sub.mk
index b07c8f939..2d2a33701 100644
--- a/core/arch/arm/plat-stm32mp1/drivers/sub.mk
+++ b/core/arch/arm/plat-stm32mp1/drivers/sub.mk
@@ -1,3 +1,3 @@
 srcs-$(CFG_STPMIC1) += stm32mp1_pmic.c
 srcs-y 	+= stm32mp1_pwr.c
-srcs-y 	+= stm32mp1_syscfg.c
+srcs-$(CFG_SYSCFG) += stm32mp1_syscfg.c
-- 
2.47.2

